### PR TITLE
Utils: Improving create_embed method to contain explicit kwargs

### PR DESCRIPTION
**Pull request by giovannitgl**
This is in reference to the issue #78 @sinus-x

### CODE REVIEW

File: core/utils.py [hunk 1]
```diff
@@ -160,36 +160,42 @@ def create_embed(
         *,
         error: bool = False,
         author: Union[discord.Member, discord.User] = None,
-        **kwargs,
+        title: str = discord.Embed.Empty,
+        description: str = discord.Embed.Empty,
```
Comment (sinus-x) [inline core/utils.py:164]: The type hints are wrong here. You are specifying just a `str`, but default is embed empty. They should be ...`: Union[str, discord.Embed.Empty] = `...

**Review by sinus-x — CHANGES_REQUESTED** [review]
Hi, thanks for picking up this issue!

I like the changes, I've just left a small note concerning the type hints.

There is also unrelated change of deleting the `gettext.py` file, please, leave the file where it was, it is important :)

If you know how to amend changes and how to do force-push with new version of the commit, please do; if you don't, add the changes as a second commit and I'll squash it while merging.

**Comment by giovannitgl** [comment]
Didn't mean to commit the delete file, sorry. Btw @sinus-x how do you run the pre-commit hooks without having an error from gettext? 
`cannot import name 'gettext' from 'gettext'`
Ever had this issue?

**Comment by sinus-x** [comment]
It's currently not possible to run it as pre-commit hook, unfortunately. `gettext` is also name of one standard library and their names are clashing, so the interpreter gaves up. We're still in alpha (and we haven't even started marking our versions yet), I'm expecting this to become relevant later in the development process.

tl;dr: The script has to be run manually for now. The part of code you've altered does not use translations, so you can ignore it, though.

**Comment by sinus-x** [comment]
And if your pre-commit tries to work with the file, you can run `git commit --no-verify` to bypass the hooks. I'll accept the PR even if it fails in the CI, the script file has been merged yesterday, so there may be some quirks.

**Comment by giovannitgl** [comment]
Alright, thanks! It's correct

File: core/utils.py [hunk 2]
```diff
@@ -160,36 +160,42 @@ def create_embed(
         *,
         error: bool = False,
         author: Union[discord.Member, discord.User] = None,
-        **kwargs,
+        title: Union[str, discord.Colour] = discord.Embed.Empty,
+        description: Union[str, discord.Colour] = discord.Embed.Empty,
+        footer: str = None,
```
Comment (sinus-x) [inline core/utils.py:165]: This type hint has to be `Optional[str]`.

File: core/utils.py [hunk 3]
```diff
@@ -160,36 +160,42 @@ def create_embed(
         *,
         error: bool = False,
         author: Union[discord.Member, discord.User] = None,
-        **kwargs,
+        title: Union[str, discord.Colour] = discord.Embed.Empty,
+        description: Union[str, discord.Colour] = discord.Embed.Empty,
```
Comment (sinus-x) [inline core/utils.py:164]: `Union[str, discord.Embed.Empty]` instead of `Union[str, discord.Colour]` :)

File: core/utils.py [hunk 4]
```diff
@@ -160,36 +160,42 @@ def create_embed(
...
+        footer: str = None,
+        color: Union[int, discord.Colour] = None,
```
Comment (sinus-x) [inline core/utils.py:166]: Since it is colour or nothing, it should be `Optional[int, discord.Colour]`.

**Review by sinus-x — CHANGES_REQUESTED** [review]
The type hints are not quite correct. But we're getting closer.

File: core/utils.py (hunk 3 above)
Comment (giovannitgl) [reply core/utils.py:164]: Oops

**Comment by giovannitgl** [comment]
Sorry for the lack of attention

File: core/utils.py [hunk 5]
```diff
@@ -160,36 +160,42 @@ def create_embed(
         *,
         error: bool = False,
         author: Union[discord.Member, discord.User] = None,
-        **kwargs,
+        title: Union[str, discord.Embed.Empty] = discord.Embed.Empty,
+        description: Union[str, discord.Embed.Empty] = discord.Embed.Empty,
+        footer: Optional[str] = None,
+        color: Optional[int, discord.Colour] = None,
```
Comment (sinus-x) [inline core/utils.py:166]: This line would throw `TypeError: Optional[t] requires a single type.` There has to be an union inside.
```py
Optional[Union[int, discord.Colour]]
```

**Review by sinus-x — CHANGES_REQUESTED** [review]
Just one tiny change. Then we'll be finished.

File: core/utils.py (hunk 5 above)
Comment (giovannitgl) [reply core/utils.py]: Thanks, haven't had the opportunity to look too in depth in python types annotations, just learnt some things!

Comment (Czechbol) [reply core/utils.py]: And we thank you for choosing to contribute to this project of ours!

It might not look like it but we appreciate it very much.

**Comment by sinus-x** [comment]
@giovannitgl The build failed, but we'll deal with that on our own, especially since you are not experienced with the type hints. We're glad you decided to help us and that you learned something new along the way.

@Czechbol LGTM, can be merged

**Review by sinus-x — APPROVED** [review]
